common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -37,12 +37,12 @@ prplmesh_platform_db_init() {
     management_mode=${1-Multi-AP-Controller-and-Agent}
     operating_mode=${2-Gateway}
 
-    mkdir -p @PLATFORM_DB_PATH_TEMP@/
+    mkdir -p /tmp/
```

Comment:
Rebase screw-up?

Reply:
no, fixing another issue. I didn't know you will be reviewing, I should have opened a draft PR. But its a good thing you review this now